packages/core/medusa-test-utils/src/module-test-runner.ts

```diff
@@ -78,7 +78,7 @@ export function moduleIntegrationTestRunner({
   const moduleOptions_: InitModulesOptions = {
     injectedDependencies: {
       [ContainerRegistrationKeys.PG_CONNECTION]: connection,
-      eventBusService: new MockEventBusService(),
+      ["eventBusModuleService"]: new MockEventBusService(),
```

Comment:
nit: ModuleRegistrationNames?

Reply:
@adrien2p the build was complaining when importing the SDK, so I left it as is for now

Reply:
oh yeah true, it should be in the utils I believe but lets think that later